app/code/community/Bolt/Boltpay/Helper/Data.php

```diff
@@ -336,4 +336,27 @@ public function getItemImageUrl($item)
 
         return (string) $image;
     }
+
+    /**
+     * Set customer session based on the quote id passed in
+     *
+     * @param $quoteId
+     */
+    public function setCustomerSessionByQuoteId($quoteId)
+    {
+        $customerId = Mage::getModel('sales/quote')->loadByIdWithoutStore($quoteId)->getCustomerId();
+        $this->setCustomerSessionById($customerId);
+    }
+
+    /**
+     * Set customer session based on the customer id passed in
+     *
+     * @param $customerId
+     */
+    public function setCustomerSessionById($customerId)
+    {
+        if ($customerId) {
+            Mage::getSingleton('customer/session')->loginById($customerId);
+        }
+    }
 }
```

Comment:
Missed new line after }

Reply:
@YevhenShBolt can you explain this a little more? I don't typically add a new line at the end of my classes. Am I missing something?

Reply:
I just noticed when i've reviewed PR that GitHub add a warning here https://github.com/BoltApp/bolt-magento1/pull/206/files. The GitHub said that there is no line at the end of a file.  

We can easily fix it:
- https://help.github.com/articles/dealing-with-line-endings/
- https://stackoverflow.com/a/46325278